libs/net/template/TemplateEngine.cs

```diff
@@ -11,6 +11,14 @@ public class TemplateEngine<T> : ITemplateEngine<T>
     where T : IRazorEngineTemplate
 {
     #region Variables
+    private readonly static string[] DEFAULT_ASSEMBLIES = new[] {
```

Comment:
I'm not a fan of hardcoding these, as we'll have to update them as we create new reports that need additional libraries.